tests/Collection/SearchIndexFunctionalTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace MongoDB\Tests\Collection;
+
+use MongoDB\Driver\Cursor;
+use MongoDB\Tests\FunctionalTestCase;
+
+use function bin2hex;
+use function getenv;
+use function random_bytes;
+use function sleep;
+
+/**
+ * Functional tests for the Atlas Search index management.
+ */
+class SearchIndexFunctionalTest extends FunctionalTestCase
+{
+    public function testIndexLifecycle(): void
+    {
+        $atlasUri = getenv('MONGODB_ATLAS_URI');
+
+        if (! $atlasUri) {
+            $this->markTestSkipped('MONGODB_ATLAS_URI environment variable is not set');
+        }
+
+        $this->manager = static::createTestManager($atlasUri);
+
```

Comment:
I'd like to check the server requirement (Atlas + version >= 7.0) or skip the test.